ui/src/app/metadata/named-references/named-reference-table/named-reference-table.component.html

```diff
@@ -48,7 +56,7 @@
         [selectAllEnabled]="selectAllEnabled"
```

Comment:
You can use [checkAll] (new property) to fix behavior with select (see at job codes to implement)
- If you select all, and you do a new search (select all is still selected)
- If you click on select all, and you unselect one, all is still selected.